variable/CMakeLists.txt

```diff
@@ -76,6 +76,11 @@ endif(DYNAMIC_LIB)
 
 add_library(${TARGET_NAME} ${LINK_TYPE} ${INC_FILES} ${SRC_FILES})
 generate_export_header(${TARGET_NAME})
+# SCIPP_EXPORT is used in macros defined in this module, but for use also in
+# other modules
+target_compile_definitions(
+  ${TARGET_NAME} PRIVATE SCIPP_EXPORT=SCIPP_VARIABLE_EXPORT
+)
```

Comment:
Not sure I understand what's going on here.

Reply:
Improved comment. Is there a cleaner way to achieve this?